client/sites-dashboard/components/sites-site-item-thumbnail.tsx

```diff
@@ -8,6 +8,8 @@ import { useSiteStatus } from '../hooks/use-site-status';
 
 const NoIcon = styled.div( {
 	fontSize: 'xx-large',
+	textTransform: 'uppercase',
```

Comment:
This is so we set a standard for the initial. We could have upper or lower depending on if the site has the title or not.